corehq/apps/hqadmin/management/commands/republish_couch_case_changes.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management import BaseCommand
+
+from dimagi.utils.chunked import chunked
+
+from casexml.apps.case.models import CommCareCase
+from corehq.doctypemigrations.continuous_migrate import _bulk_get_revs
+
+
+class Command(BaseCommand):
+    """
+    Republish case changes
+            $ ./manage.py republish_couch_case_changes <DOMAIN> <case_ids.txt>
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+        parser.add_argument('case_ids_file')
+
+    def handle(self, domain, case_ids_file, *args, **options):
+        self.domain = domain
+        case_ids = self.get_case_ids(case_ids_file)
+        self.publish_cases(domain, case_ids)
+
+    def get_case_ids(self, case_ids_file):
+        with open(case_ids_file, 'r') as f:
+            lines = f.readlines()
+            return [l.strip() for l in lines]
+
+    def publish_cases(self, domain, case_ids):
+        from corehq.apps.hqcase.management.commands.backfill_couch_forms_and_cases import (
+            publish_change, create_case_change_meta)
```

Comment:
nit: could consider pulling these out into a utility module if they're going to be used outside the defining command.